owncloudData/src/test/java/com/owncloud/android/data/file/repository/OCFileRepositoryTest.kt

```diff
@@ -229,14 +229,14 @@ class OCFileRepositoryTest {
 
     @Test
     fun `remove file - ok`() {
-        every { remoteFileDataSource.removeFile(any(), any()) } returns Unit
+        every { remoteFileDataSource.deleteFile(any(), any()) } returns Unit
         every { localStorageProvider.deleteLocalFile(any()) } returns true
 
-        ocFileRepository.removeFile(listOfFilesToRemove = listOfFileToRemove, removeOnlyLocalCopy = false)
+        ocFileRepository.deleteFile(listOfFilesToDelete = listOfFileToRemove, removeOnlyLocalCopy = false)
```

Comment:
I see that we use a singular name when deleting a list, so I would update it to `deleteFiles` by the way 👍

Reply:
True, I didn't realise! Change applied 👍